homeassistant/components/acmeda/sensor.py

```diff
@@ -0,0 +1,74 @@
+"""Support for Acmeda Roller Blind Batteries."""
+import asyncio
+
+import aiopulse
+
+from homeassistant.const import (
+    ATTR_BATTERY_LEVEL,
+    DEVICE_CLASS_BATTERY,
+    UNIT_PERCENTAGE,
+)
+from homeassistant.helpers import entity
+from homeassistant.helpers.dispatcher import async_dispatcher_connect
+from homeassistant.helpers.icon import icon_for_battery_level
+
+from .base import AcmedaBase
+from .const import ACMEDA_HUB_UPDATE
+from .helpers import update_entities
+
+
+async def async_setup_entry(hass, config_entry, async_add_entities):
+    """Set up the Acmeda Rollers from a config entry."""
+    update_lock = asyncio.Lock()
+    current = {}
+
+    async def async_update():
+        async with update_lock:
+            await update_entities(
+                hass, AcmedaBattery, config_entry, current, async_add_entities
+            )
+
+    async_dispatcher_connect(hass, ACMEDA_HUB_UPDATE, async_update)
+
+
+class AcmedaBattery(AcmedaBase, entity.Entity):
+    """Representation of a Acmeda cover device."""
+
+    device_class = DEVICE_CLASS_BATTERY
+    unit_of_measurement = UNIT_PERCENTAGE
+
+    def __init__(self, hass, roller: aiopulse.Roller):
+        """Initialize the roller."""
+        super().__init__(hass, roller)
+
+    @property
+    def name(self):
+        """Return the name of roller."""
+        return super().name + " Battery"
```

Comment:
```suggestion
        return f"{super().name} Battery"
```

The Home Assistant standard is to use f-strings.

Reply:
Resolved.